extensions/podman/packages/extension/src/extension.ts

```diff
@@ -2278,3 +2292,24 @@ export function updateWSLHyperVEnabledContextValue(value: boolean): void {
     extensionApi.context.setValue(WSL_HYPERV_ENABLED_KEY, value);
   }
 }
+
+async function initializeCertificateDetection(telemetryLogger: extensionApi.TelemetryLogger): Promise<void> {
+  certificateDetectionService = new CertificateDetectionService(telemetryLogger, {
+    enableTelemetry: true,
+  });
+
+  // Initial detection
+  await certificateDetectionService.detectCustomCertificates();
+
+  // Set up periodic detection (every 24 hours)
```

Comment:
do we need this check every 24hour or just once per launch ?

Reply:
Theoretically, user can hold podman desktop opened for a long time and user can configure custom registry during this period, and it would be nice to handle the registry configuration changes.